liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -178,11 +177,10 @@ void SourceReferenceFormatter::printSourceLocation(SourceReference const& _ref)
 
 void SourceReferenceFormatter::printExceptionInformation(SourceReferenceExtractor::Message const& _msg)
 {
-	// exception header line
-	optional<Error::Severity> severity = Error::severityFromString(_msg.severity);
-	errorColored(severity) << _msg.severity;
+	Error::Type type = _msg.type;
+	errorColored(Error::errorSeverity(_msg.type)) << Error::formatErrorType(_msg.type);
```

Comment:
This is not equivalent to the old version. Now it prints the exact type while it used to print severity. It should be this instead:
```suggestion
	errorColored(Error::errorSeverity(_msg.type)) << Error::formatErrorSeverity(Error::errorSeverity(_msg.type));
```

The result is that you get specific error types in all those command-line tests. They should not change. I think we only print severity in the CLI on purpose and I'd not change that in this PR.

Reply:
Updated in the latest push.